README.md

```diff
@@ -51,4 +51,12 @@ $ helm install devfile-registry ./deploy/chart/devfile-registry \
 	--set devfileIndex.tag=latest
 ```
 
-For more information on the Helm chart, consult [its readme](deploy/chart/devfile-registry/README.md).
\ No newline at end of file
+For more information on the Helm chart, consult [its readme](deploy/chart/devfile-registry/README.md).
+
+## Contributing
+
+Please see our [contributing.md](./CONTRIBUTING.md).
+
+## License
```

Comment:
can add the following at the top instead
```
<div id="header">

[![Apache2.0 License](https://img.shields.io/badge/license-Apache2.0-brightgreen.svg)](LICENSE)
</div>
```

Reply:
Done.